### ISSUE TITLE
Adding generated files to .gitignore

### ISSUE THREAD

**Issue by oneself** [eiichiroi/autotools-unittest#1]
Hi,

Starting from a cleanly cloned repo, and going through the steps in the README file, I get a number of generated files that are not in .gitignore.  I was wondering if these have been left out intentionally.  Should these be checked into git when creating a new project or should these be added to .gitignore?  (I'd be happy to create a pull request for this).

```
$ ./autogen.sh
$ ./configure --enable-coverage
$ make check
$ make coverage
$ git status
# Untracked files:
#       lib/gmock/.dirstamp
#       m4/libtool.m4
#       m4/ltoptions.m4
#       m4/ltsugar.m4
#       m4/ltversion.m4
#       m4/lt~obsolete.m4
#       src/autotools-unittest/.dirstamp
#       test-suite.log
#       test/.dirstamp
#       test/sample_test.log
#       test/sample_test.trs
```

Thank you, 

p.s. I love this project.  It saved me a bunch of time.

**Comment by eiichiroi** [comment]
Oh, thanks. I will fix this issue before long.

**Comment by eiichiroi** [comment]
I'm sorry I was left long... I have fixed in 6c3f808.
